lib/conditionals.md

```diff
@@ -5,8 +5,10 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A conditional is a statement which sets the parameters for which parts of the code will be executed.
```

Comment:
Can you say more about what "setting the parameters" means? "Parameter" is a word that gets used itself in programming so I want to make sure I fully understand your answer.